core/src/main/scala/org/scalawag/jibe/mandate/command/PackageCommands.scala

```diff
@@ -0,0 +1,19 @@
+package org.scalawag.jibe.mandate.command
+
+import org.scalawag.jibe.mandate.Package
+
+/** Returns true eturns true if the package is installed at the specified version.
+  * If no version is specified, only returns true if it is the most recent version.
+  */
+
+case class IsPackageInstalled(pkg: Package) extends BooleanCommand
+
+/** Returns true if the package is installed at the specified version or the most recent if not specified.
+  */
+
+case class InstallPackage(pkg: Package) extends UnitCommand
+
+/** Runs apt-get update if the cache is old enough
+  */
+
+case class UpdateAptGet(ageInHours: Int) extends UnitCommand
```

Comment:
But I'll need to call toSeconds on the refreshInterval before it's passed into the shell script. I'm still trying to figure out where I get to do that.